pkg/util/kubernetes/const.go

```diff
@@ -35,4 +35,55 @@ const (
 
 	// KubeNodeRoleTagName is the role label tag name
 	KubeNodeRoleTagName = "kube_node_role"
+
+	// PodKind represents the Pod object kind
+	PodKind = "Pod"
+	// DeploymentKind represents the Deployment object kind
+	DeploymentKind = "Deployment"
+	// ReplicaSetKind represents the ReplicaSet object kind
+	ReplicaSetKind = "ReplicaSet"
+	// ReplicationControllerKind represents the ReplicaSetController object kind
+	ReplicationControllerKind = "ReplicationController"
+	// StatefulSetKind represents the StatefulSet object kind
+	StatefulSetKind = "StatefulSet"
+	// DaemonSetKind represents the DaemonSet object kind
+	DaemonSetKind = "DaemonSet"
+	// JobKind represents the Job object kind
+	JobKind = "Job"
+	// CronJobKind represents the CronJob object kind
+	CronJobKind = "CronJob"
+	// ServiceKind represents the ServiceKind object kind
+	ServiceKind = "Service"
+
+	// PodTagName represents the pods tag name
+	PodTagName = "pod_name"
+	// DeploymentTagName represents the Deployment tag name
+	DeploymentTagName = "kube_deployment"
+	// ReplicaSetTagName represents the ReplicaSet tag name
+	ReplicaSetTagName = "kube_replica_set"
+	// ReplicaSetControllerTagName represents the ReplicaSetController tag name
+	ReplicaSetControllerTagName = "kube_replication_controller"
```

Comment:
Nit - can we keep it as `ReplicationController..` to avoid confusion?

Reply:
good catch!